### ISSUE TITLE
Show "activate the X-Ray Goggles" module if goggles aren't activated 

### ISSUE THREAD

**Issue by kristinashu** [mozilla/goggles.mozilla.org#289]
When on the [FTU activity page](https://goggles.mozilla.org/ftu) and part way through the tutorial, if I esc out of goggles then the initial "step 1: activate the X-Ray Goggles" should appear. But right now it goes to the "Remix the Images" box.

cc @Pomax

**Comment by Pomax** [comment]
Yeah there's no check to see whether you closed goggles in the mean time atm, good idea to add that in.

**Comment by kristinashu** [comment]
It would also be better if it didn't go back to the "Remix the images" step when you're on the "Remix the Words" step

**Comment by Pomax** [comment]
I'm not sure I know exactly *why* it goes back to that step, but should be revealed solving this.
